Servers/WindowServer/WSMenu.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ *                          Shannon Booth <shannon.ml.booth@gmail.com>
```

Comment:
Sorry, I should have been more clear. Please also include the year like so:
```suggestion
 * Copyright (c) 2020, Shannon Booth <shannon.ml.booth@gmail.com>
```

Reply:
Oh okay :). Never worked with copyright stuff before, thanks :)